README.md

```diff
@@ -1,36 +1,162 @@
-<h1> !!!This repo is under development and NOT stable!!! </h1>
+#### **_this repo is under development and not stable!_**
 
 # napari
+
+### multi-dimensional image viewer for python
+
 [![License](https://img.shields.io/pypi/l/napari.svg)](https://github.com/napari/napari/raw/master/LICENSE)
 [![Build Status](https://api.cirrus-ci.com/github/Napari/napari.svg)](https://cirrus-ci.com/napari/napari)
 [![Python Version](https://img.shields.io/pypi/pyversions/napari.svg)](https://python.org)
 [![PyPI](https://img.shields.io/pypi/v/napari.svg)](https://pypi.org/project/napari)
 [![PyPI - Downloads](https://img.shields.io/pypi/dm/napari.svg)](https://pypistats.org/packages/napari)
 [![Development Status](https://img.shields.io/pypi/status/napari.svg)](https://github.com/napari/napari)
 
----
+**napari** is a fast, interactive, multi-dimensional image viewer for Python. It's designed for browsing, annotating, and analyzing large multi-dimensional images. It's built on top of `PyQt` (for the GUI), `vispy` (for performant GPU-based rendering), and the scientific Python stack (`numpy`, `scipy`).
 
-napari is a fast multi-dimensional image viewer for Python.
+We're developing **napari** in the open! But the project is in a **pre-alpha** stage. You can follow progress on this repository, test out new versions as we release them, and contribute ideas and code. Expect **breaking changes** from patch to patch.
 
-This project is in the **pre-alpha** stage; **expect breaking changes** to be made from patch to patch.
+## installation
 
-## Installation
+**napari** can be installed on most Mac OS X and Linux systems with Python 3.6 or 3.7 by calling 
 
-napari can be installed on most operating systems with `pip install napari`.
+```sh
+$ pip install napari
+```
 
-### Downloading OpenGL
+(We're working on adding Windows support.)
 
-Windows and Linux users may need to install the [Mesa OpenGL](https://www.mesa3d.org/) library.
+To install from the master branch on Github use
 
-`vispy` provides an easily-downloadable (but out-of-date) Windows version for [64-](https://github.com/vispy/demo-data/raw/master/mesa/opengl32_mingw_64.dll) and [32-bit](https://github.com/vispy/demo-data/raw/master/mesa/opengl32_mingw_64.dll) operating systems.
+```sh
+$ pip install git+https://github.com/napari/napari
+```
 
-Linux users can complete the installation process from the command line:
+To clone the repository locally and install in editable mode use
 
 ```sh
-$ sudo apt update
-$ sudo apt install libgl1-mesa-glx
+$ git clone https://github.com/napari/napari.git
+$ cd napari
+$ pip install -e .
+```
+
+Note that many of our examples use `data` from `skimage` but `skimage` is otherwise not a dependnecy, so to run all the examples you should call
+
+```sh
+$ pip install skimage
```

Comment:
I know it's annoying but: 
```suggestion
$ pip install scikit-image
```

Reply:
"so to run all the examples you should call" is a bit misleading, can you say something like 
"so to run all the examples you should need to install skimage via"

Reply:
i just saw the thread was about to say this XD